pkg/apis/kubeone/validation/validation.go

```diff
@@ -345,35 +336,6 @@ func ValidateKubernetesSupport(c kubeoneapi.KubeOneCluster, fldPath *field.Path)
 	return allErrs
 }
 
-func ValidateContainerRuntimeConfig(cr kubeoneapi.ContainerRuntimeConfig, versions kubeoneapi.VersionConfig, fldPath *field.Path) field.ErrorList {
-	allErrs := field.ErrorList{}
-
-	allCRs := []interface{}{
-		cr.Docker,
-		cr.Containerd,
-	}
-
-	var found bool
-	for _, x := range allCRs {
-		if !reflect.ValueOf(x).IsNil() {
-			if found {
-				allErrs = append(allErrs, field.Invalid(fldPath, x, "only 1 container runtime can be activated"))
-			}
-			found = true
-		}
-	}
```

Comment:
I'm wondering if leaving such a function might be useful if we decide to add support for another CRI (e.g. CRI-O)